Uplift/Controllers/GymDetailViewController.swift

```diff
@@ -49,6 +50,10 @@ class GymDetailViewController: UIViewController {
         super.init(nibName: nil, bundle: nil)
         self.gymDetail = GymDetail(gym: gym)
 
+        for facility in gymDetail.facilities where facility.name == "Fitness Center" {
```

Comment:
is "Fitness Center" used in more than one file? I feel like it probably is and we should make it a global constant

Reply:
only all uppercased